src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -442,6 +442,8 @@ const InvoiceForm = ({
                           label={<FormattedMessage id="ui-invoice.invoice.currentExchangeRate" />}
                           exchangeFrom={filledCurrency}
                           exchangeTo={systemCurrency}
+                          setExchangeRateEnabled={setExchangeRateEnabled}
```

Comment:
don't think it's right approach

imo you should use context

Reply:
but anyway I need to use setState to change some values from the Child component

Reply:
making parent component dependent from child is not the best idea (especially without redux)

why not to move exchange rate loading to another level?

Reply:
this makes sense, especially if you can say without specific flags that input is required or enabled.